.storybook/stories/DataFiltering/TableFilteringMedium.tsx

```diff
@@ -96,7 +156,48 @@ export const TableFilteringMedium = ({
 						<DataTableSelectAllCheckbox />
 					</Box>
 				)}
-				<DataTable selectable={selectable} headingId={headingId} />
+
+				<DataTable
+					selectable={selectable}
+					headingId={headingId}
+					onOpenDrawer={onOpenDrawer}
+				/>
+
+				<Drawer
+					actions={
+						<ButtonGroup>
+							<Button type="submit" form="form-id" onClick={onSubmitForm}>
+								Save changes
+							</Button>
+
+							<Button variant="tertiary" onClick={closeDrawer}>
+								Cancel
+							</Button>
+						</ButtonGroup>
+					}
+					isOpen={isDrawerOpen}
+					onClose={closeDrawer}
+					title={`Edit ${
+						currentItem?.businessName ? currentItem.businessName : 'Business'
+					}`}
+				>
+					<form id="form-id" ref={formRef}>
```

Comment:
You don't appear to be doing anything with this ref?

Reply:
Oh yeah, we don't need this now. It was to get the data from the form but I'm just taking it from state instead, which isn't great but it'll do for now.

- [ ] no form ref